File: tkpdreputation/src/main/java/com/tokopedia/tkpd/tkpdreputation/inbox/domain/model/InboxReputationDomain.java
package com.tokopedia.tkpd.tkpdreputation.inbox.domain.model;

import java.util.List;

/**
 * @author by nisie on 8/14/17.
 */

public class InboxReputationDomain {

    private List<InboxReputationItemDomain> inboxReputation = null;

    private NotificationDomain notification;

    private PagingDomain paging;

    public InboxReputationDomain(List<InboxReputationItemDomain> inboxReputation,
                                 NotificationDomain notification, PagingDomain paging) {
        this.inboxReputation = inboxReputation;
        this.notification = notification;
        this.paging = paging;
    }

    public List<InboxReputationItemDomain> getInboxReputation() {
        return inboxReputation;
    }

    public NotificationDomain getNotification() {
        return notification;
    }

    public PagingDomain getPaging() {
        return paging;
    }
}

File: features/merchant/review_inbox/src/main/java/com/tokopedia/review/feature/inbox/buyerreview/domain/model/InboxReputationDomain.kt
package com.tokopedia.review.feature.inbox.buyerreview.domain.model

/**
 * @author by nisie on 8/14/17.
 */
data class InboxReputationDomain (
    val inboxReputation: List<InboxReputationItemDomain> = listOf(),
    val paging: PagingDomain = PagingDomain()
)